### PR TITLE
Add support for host stress events

### PR DESCRIPTION
**Pull request by sathnaga**
Add support for host stress events, which would help
us to run multiple events in host in parallel to events for
guest and stress.

Signed-off-by: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by sathnaga** [comment]
Depends on: https://github.com/avocado-framework/avocado/pull/2697

File: virttest/utils_stress.py [hunk 1]
```diff
@@ -158,3 +163,27 @@ def vm_stress_events(self, event, vm):
...
                 raise NotImplementedError
+
+    def host_stress_event(self, event):
+        """
+        Host Stress events
+
+        :param event: event name
+        """
+        for itr in range(self.host_iterations):
+            if "cpufreqgovernor" in event:
```
Comment (balamuruhans) [inline virttest/utils_stress.py]: for better readability can we have the name like `cpu_freq_governor` for this and other events too.

File: virttest/utils_stress.py [hunk 2]
```diff
@@ -158,3 +163,27 @@ def vm_stress_events(self, event, vm):
...
+                cpu.set_cpufreq_governor()
+                logging.debug("Current governor: %s", cpu.get_cpufreq_governor())
+                time.sleep(self.event_sleep_time)
+            elif "cpuidlestate" in event:
+                idlestate = cpu.get_cpuidle_state()
+                cpu.set_cpuidle_state()
+                time.sleep(self.event_sleep_time)
+                cpu.set_cpuidle_state(setstate=idlestate)
```
Comment (balamuruhans) [inline virttest/utils_stress.py]: how do we make sure the cpu idle state is set or not ? can we do get_cpuidle_state() after setting to validate ?

File: virttest/utils_stress.py [hunk 3]
```diff
@@ -158,3 +163,27 @@ def vm_stress_events(self, event, vm):
...
+            elif "cpuoffline" in event:
+                processor = self.host_cpu_list[random.randint(0, cpu.online_cpus_count()-1)]
+                cpu.offline(processor)
+                time.sleep(self.event_sleep_time)
+                cpu.online(processor)
```
Comment (balamuruhans) [inline virttest/utils_stress.py]: same here

File: virttest/utils_stress.py (hunk 2 above)
Comment (sathnaga) [reply virttest/utils_stress.py]: validation will make the stress events heavier, there is check in the `set_cpuidle_state` function itself and that would suffice.

File: virttest/utils_stress.py (hunk 3 above)
Comment (sathnaga) [reply virttest/utils_stress.py]: same as above.

File: virttest/utils_stress.py (hunk 1 above)
Comment (sathnaga) [reply virttest/utils_stress.py]: Sure, I can change it

**Pull request merged**